recipes/cassandra-driver/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "cassandra-driver" %}
+{% set version = "3.8.1" %}
+{% set sha256 = "" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - numpy
```

Comment:
I *think* this needs to be `numpy x.x` here and in run requirements because numpy is being imported in [numpy_parser.pyx](https://github.com/datastax/python-driver/blob/master/cassandra/numpy_parser.pyx#L37). Am I correct here @jakirkham, @ocefpaf or @msarahan ?